config/vendor-api-0.5.0.yml

```diff
@@ -304,6 +304,7 @@ components:
           type: string
           description: The status of this application
           enum:
+            - unsubmitted
```

Comment:
Good catch - can we add an extra test to https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/services/application_state_change_spec.rb to make sure the valid states are the same as this enum?

Reply:
Sure I can add that in another PR :)